src/ConfigCat.Cli/ConfigCat.Cli.csproj

```diff
@@ -11,24 +11,41 @@
     <AssemblyName>configcat</AssemblyName>
     <AssemblyTitle>ConfigCat CLI</AssemblyTitle>
     <InvariantGlobalization>true</InvariantGlobalization>
-    <PublishTrimmed>true</PublishTrimmed>
-    <TrimmerDefaultAction>link</TrimmerDefaultAction>
-    <TrimmerRemoveSymbols>true</TrimmerRemoveSymbols>
-    <IncludeAllContentForSelfExtract>true</IncludeAllContentForSelfExtract>
-    <IncludeNativeLibrariesForSelfExtract>true</IncludeNativeLibrariesForSelfExtract>
-    <PublishReadyToRun>true</PublishReadyToRun>
     <Version>0.0.0</Version>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Content Include="..\..\test\integ.ps1" Link="integ.ps1">
-    </Content>
-  </ItemGroup>
+  <Choose>
+    <When Condition="'$(_IsPacking)' == 'true'">
+      <!-- Setup for .NET CLI tool build -->
+      <PropertyGroup>
+        <PackAsTool>true</PackAsTool>
+        <PackageId>ConfigCat.Cli</PackageId>
```

Comment:
A more "command-like" id would be better here, maybe `configcat-cli`? 
So the install would look like: `dotnet tool install configcat-cli`